account_brand/models/res_partner_account_brand.py

```diff
@@ -19,7 +19,7 @@ class ResPartnerAccountBrand(models.Model):
         comodel_name="account.account",
         string="Account",
         required=True,
-        domain="[('account_type', 'in', ('liability_payable', 'asset_receivable'))]",
+        domain="[('account_type', '=', account_type)]",
```

Comment:
Is domain evaluated dynamically? I think this is evaluated statically, so, if you change `account_type` field, the list is not adapted, correct ?